internal/controller/sync.go

```diff
@@ -352,19 +357,127 @@ func (r *Route) buildNextCR(ctx context.Context, route *routev1.Route, revision
 			uriSans = append(uriSans, ur)
 		}
 	}
+	var emailAddresses []string
+	if metav1.HasAnnotation(route.ObjectMeta, cmapi.EmailsAnnotationKey) {
+		emailAddresses = strings.Split(route.Annotations[cmapi.EmailsAnnotationKey], ",")
+	}
+	var organizations []string
+	if metav1.HasAnnotation(route.ObjectMeta, cmapi.SubjectOrganizationsAnnotationKey) {
+		subjectOrganizations, err := cmutil.SplitWithEscapeCSV(route.Annotations[cmapi.SubjectOrganizationsAnnotationKey])
+		organizations = subjectOrganizations
+
+		if err != nil {
+			r.log.V(1).Error(err, "the organizations annotation is invalid",
+				"object", route.Namespace+"/"+route.Name, cmapi.SubjectOrganizationsAnnotationKey,
+				route.Annotations[cmapi.SubjectOrganizationsAnnotationKey])
+			r.eventRecorder.Event(route, corev1.EventTypeWarning, ReasonInvalidValue, "annotation "+cmapi.SubjectOrganizationsAnnotationKey+": "+route.Annotations[cmapi.SubjectOrganizationsAnnotationKey]+" value is malformed")
+			return nil, err
+		}
+	}
+	var organizationalUnits []string
+	if metav1.HasAnnotation(route.ObjectMeta, cmapi.SubjectOrganizationalUnitsAnnotationKey) {
+		subjectOrganizationalUnits, err := cmutil.SplitWithEscapeCSV(route.Annotations[cmapi.SubjectOrganizationalUnitsAnnotationKey])
+		organizationalUnits = subjectOrganizationalUnits
+
+		if err != nil {
+			r.log.V(1).Error(err, "the organizational units annotation is invalid",
```

Comment:
`log.V(1).Error` may confuse ourselves, we may think that `-v=0` won't show that message. In reality, `log.V(1).Error` acts like `log.V(0).Error`. I suggest that we remove `V(1)` to avoid any confusion on our side (by "our" I mean the developers of openshift-routes):



```suggestion
			r.log.Error(err, "the organizational units annotation is invalid",
```

Reply:
Let's do that later too.